app/Jobs/ApplicationDeploymentJob.php

```
@@ -1358,16 +1464,19 @@ private function prepare_builder_image()
         // Get user home directory
         $this->serverUserHomeDir = instant_remote_process(['echo $HOME'], $this->server);
         $this->dockerConfigFileExists = instant_remote_process(["test -f {$this->serverUserHomeDir}/.docker/config.json && echo 'OK' || echo 'NOK'"], $this->server);
+
+        $env_flags = $this->generate_docker_env_flags_for_secrets();
```

PHPMD (2.15.0)

The variable $env_flags is not named in camelCase. (undefined)

(CamelCaseVariableName)